script/release_utils/check_branch.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
Maybe adding a small descriptive about why this script is necessary would be nice to have.